.github/workflows/release-contracts.yml

```diff
@@ -0,0 +1,51 @@
+name: Release contracts
+
+permissions:
+  contents: write
+
+on:
+  push:
+    tags:
+      - 'v*'
+    branches:
+      - main
+      - ci/release-contracts
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    container: cosmwasm/workspace-optimizer:0.12.11
```

Comment:
i'm a little nervous that this will become out of date when a new workspace-optimizer release is made. ideally we'd be able to synchronize this with the version used in the `workspace-optimize`.

i wonder if we could integrate this with our existing [integrationt test workflow](https://github.com/DA0-DA0/dao-contracts/blob/main/.github/workflows/integration_tests.yml#L61) so to keep that in sync?

for example, the `just integration-test` command there builds the contracts using the `just workspace-optimize` command, so if we moved the upload step here over to the job i think this would get us uploads, and make sure our versions are staying consistent.

Reply:
The main issue with `just workspace-optimize` is that it's running a docker container, it's would require to use docker in docker to build...
It's didn't feel an appropriate solution.

Maybe we can use ` cw-optimizoor`, but i believe `cosmwasm/workspace-optimizer` is the prefered tool for compiling, specially on a tag.


IMHO, having the tag is not a big problem... or we can use `:latest` ... but i doubt it's a good solution

Reply:
talking with @NoahSaso, having `.wasm` files available from CI is useful and i see the problem with docker within docker.